fastfield_codecs/src/main.rs

```diff
@@ -91,6 +91,10 @@ pub fn get_codec_test_data_sets() -> Vec<(Vec<u64>, &'static str)> {
         .collect::<Vec<_>>();
     data_and_names.push((data, "Almost monotonically increasing"));
 
+    let data = (1000..=200_000_u64)
+        .map(|_| rand::random::<u8>() as u64)
```

Comment:
Can we seed that?

Reply:
I will. But the the results is already very stable on 200 000 values